test/pstl_offload/headers/headers.standard/strings/cstring.pass.cpp

```diff
@@ -0,0 +1,16 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <cstring>
+#include "support/utils.h"
+
+int main() {
+    [[maybe_unused]] auto ptr = &std::strcpy;
```

Comment:
not 100% sure this operation is allowed.

Reply:
Changed